core/src/com/unciv/logic/map/MapGenerator.kt

```diff
@@ -653,6 +665,38 @@ class MapGenerator(val ruleset: Ruleset) {
             return Perlin.noise3d(worldCoords.x.toDouble(), worldCoords.y.toDouble(), seed, nOctaves, persistence, lacunarity, scale)
         }
 
+        private fun getDistortedPerlinNoise(tile: TileInfo, seed: Double,
```

Comment:
A perlin noise which uses multiple perlin noise to change its parameters. Lacunarity, persistence, and x and y displacements are modified using perlin noise